api/nodes/views.py

```diff
@@ -1558,6 +1559,30 @@ def create(self, *args, **kwargs):
         return ret
 
 
+class NodeSubjectsList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, NodeMixin):
+    """The documentation for this endpoint can be found [here](https://developer.osf.io/#operation/nodes_subjects_list).
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        ContributorOrPublic,
+        ExcludeWithdrawals,
+    )
+
+    required_read_scopes = [CoreScopes.NODE_BASE_READ, CoreScopes.SUBJECTS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+    serializer_class = TaxonomySerializer
+
+    model = Subject
+    view_category = 'nodes'
+    view_name = 'node-subjects'
+
+    ordering = ('-id',)
+
+    def get_queryset(self):
+        return self.get_node().subjects.all()
```

Comment:
Do we also need a self subjects view for modifying subjects? Would allow subjects to be removed entirely.

Reply:
Being able to remove subjects seems important, so I think yes?

Reply:
okay - this doesn't exist for Preprint subjects, for example, I believe. You can replace the entire list through updating, but we don't have functionality to remove subjects entirely.  But I can add these endpoints.

Reply:
Subjects are required for preprints, right? They're optional for registrations, so it is new functionality.

Reply:
I guess that implies validation on preprints to ensure > 0 subjects...

Reply:
yes, good reminder to make sure preprints behavior is still enforced, even as other endpoints are added.